app/controllers/api/v1/rooms_configurations_controller.rb

```diff
@@ -19,7 +19,7 @@
 module Api
   module V1
     class RoomsConfigurationsController < ApiController
-      before_action only: %i[index] do
+      before_action do
```

Comment:
No big deal - we settled to explicitly allow each action for more security. 
So you could add show instead.
`before_action only: %i[index show]`